gradle.properties

```diff
@@ -319,4 +319,4 @@ xercesImplVersion=2.12.2
 xmlApisVersion=1.0.b2
 
 # sync with Tika/POI
-xmlbeansVersion=5.2.0
+xmlbeansVersion=5.3.0
```

Comment:
nit: On develop, when running gradle commands I'm seeing the XMLBeans version get logged when I don't think i was previously. Here is an example running pickPg:

<img width="750" alt="image" src="https://github.com/user-attachments/assets/df5507aa-365e-455f-8f47-304157f5a2bc" />

Is this something we're logging? Is it useful? If not, could it be removed?

Reply:
@labkey-nicka Yes, we're logging the XMLBeans version in the Gradle Plugins and the server. As the PR comment suggests, we're trying to track down what seems like a version discrepancy between build time and run time. We'll remove the logging once we successfully upgrade.